admin/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+# Function to install a file whose name ends in .template
+function(install_template_file)
```

Comment:
does this function behave correctly if you call it with a non-template file name?

Reply:
What does "correctly" mean here? If it means the file will still be copied without error if it does not have `.template` in the name, then yes. The `string(REPLACE ...)` will do nothing if a match is not found.